src/main/java/com/faforever/server/config/ServerProperties.java

```diff
@@ -81,6 +82,11 @@
   }
 
   @Data
+  public static class Ladder {
+    private Integer season;
```

Comment:
Can you add documentation on what this configuration property does? Sounds like we need to reconfigure and restart the server every time the season changes. In which case it should rather be a runtime configuration, i.e. read from the database.

Reply:
I assume changing leader seasons only with big changes coming along -> server restart implied.
We could also go for yearly ladder and use the current date instead.
As of now, I would not add a configuration table for a single value.